src/panels/setupGranitePage.ts

```diff
@@ -355,19 +355,22 @@ export class SetupGranitePage {
         embeddingsModelId: graniteConfiguration.embeddingsModelId ?? 'none',
       });
     } catch (error: any) {
-      //if error is CancellationError, then we can ignore it
-      if (error instanceof CancellationError || error?.name === "Canceled") {
-        return;
-      }
```

Comment:
No you need to keep it or else you'll get an error popup. User cancelled on purpose, so no popup is necessary IMO.
<img width="537" alt="Screenshot 2024-10-01 at 18 29 46" src="https://github.com/user-attachments/assets/3a3b109f-f7e7-4eeb-a5b3-fb243c67d028">